src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const correctedRobots = {};
+
+  for (const key in robot) {
+    correctedRobots[robot[key]] = key;
+  }
+
+  if (Object.keys(correctedRobots).length !== Object.keys(robot).length) {
```

Comment:
too hard to read. make variables. but you actually do not need to use this. return null from the for loop